README.md

```diff
@@ -661,183 +661,40 @@ correctness, by implementing GC synchronously, as part of the delete request. To
 the best of my knowledge, there's nothing in the description that _requires_ GC
 to occur asynchronously.
 
```

Comment:
Our reading of the algorithms lead us to believe the GC should merely be an out-of-band process that does not block function replication or latency-critical threads. Its only requirement is all N acceptors respond that the tombstone is cleared, and that all N acceptors had written a tombstone before the GC begins.